packages/client/src/recentlyViewed/types/recentlyViewedProducts.types.ts

```diff
@@ -0,0 +1,17 @@
+export type RecentlyViewedProducts = RecentlyViewedProductsPaginationData &
+  RecentlyViewedProductsEntriesData;
+
+export type RecentlyViewedProductsPaginationData = {
+  number?: number;
```

Comment:
Question: Is there any case when these options are not available? I did a little test here and they seem to be always defined.

Reply:
Source updated, can you check again.